src/include/templates/llm_map_prompt_template.hpp

```diff
@@ -1,7 +1,7 @@
-#ifndef LF_MAP_PROMPT_TEMPLATE_H
-#define LF_MAP_PROMPT_TEMPLATE_H
+#ifndef LLM_MAP_PROMPT_TEMPLATE_H
+#define LLM_MAP_PROMPT_TEMPLATE_H
 
-constexpr auto lf_map_prompt_template = R"(
+constexpr auto llm_map_prompt_template = R"(
 {{prompts}}
 
 {
```

Comment:
Let's fix this while we are at it.

Add ### INPUT as a header right before this code.
Let's switch from JSON to markdown so that we avoid repeating the keys for every single row and reduce token usage.